charts/terraform-pod/README.md

```diff
@@ -0,0 +1,189 @@
+
+# terraform-pod
+
+## `license`
+```
+          ,-.
+ ,     ,-.   ,-.
+/ \   (   )-(   )
+\ |  ,.>-(   )-<
+ \|,' (   )-(   )
+  Y ___`-'   `-'
+  |/__/   `-'
+  |
+  |
+  |    -hi-
+__|_____________
+
+/* Copyright (C) Saritasa,LLC - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ * Written by Saritasa Devops Team, April 2022
+ */
+
+```
+
+## `chart.deprecationWarning`
+
+## `chart.name`
+
+terraform-pod
+
+## `chart.version`
+
+![Version: 0.0.7](https://img.shields.io/badge/Version-0.0.7-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.3.5](https://img.shields.io/badge/AppVersion-1.3.5-informational?style=flat-square)
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Saritasa |  | <https://www.saritasa.com/> |
+
+## `chart.description`
+
+A Helm chart for running infra-dev-aws solutions
+
+## Install the chart
+
+Install the chart:
+
+```
+helm repo add saritasa https://saritasa-nest.github.io/saritasa-devops-helm-charts/
+```
+
+## Use
+
+### Simple case (infra-dev-aws)
+
+```sh
+helm upgrade --install CLIENT saritasa/terraform-pod \
+  --namespace terraform \
+  --set terraform.client=CLIENT \
+  --set image.tag=1.3.5 \
+  --set github.repository=saritasa-nest/CLIENT-infra-dev-aws \
+  --set github.branch=feature/branch \
+  --set github.username=YOUR-GITHUB-USERNAME \
+  --set github.email=YOUR-GITHUB-EMAIL \
+  --set gitCryptKey=$(base64 -w 0 git-crypt-key) \
```

Comment:
It doesn't work for me, on mac I have no parameter -w

Reply:
@SergeyDeminSaritasa 
let me know if you get this working with something like

```sh
--set gitCryptKey=$(base64 git-crypt-key | tr -d \\n)
```

Reply:
@SergeyDeminSaritasa ping

Reply:
I need -i argument 
![image](https://user-images.githubusercontent.com/62735583/209064635-a42675a7-dd58-4aaa-ac0c-1a01a5bad83e.png)

Reply:
@SergeyDeminSaritasa question was different - let me know if command above worked for you, so we can use it as default in the Makefile target.

Reply:
I don't have Mac on me today :) do you have `uname` command in there? And if so - what is the output for you for 

```
$ uname
````

Reply:
No, command does not work for me, cause "base64 git-crypt-key" need additional argument.
`➜ uname
Darwin`

As I showed on last string in terminal screen, for me worked command
`--set gitCryptKey=$(base64 -i git-crypt-key | tr -d \\n)`
But, the result of this - I can't apply with helm, cause got error "unexpected end of string".

Reply:
solved by
`openssl base64 -in git-crypt-key | tr -d '\n'`